MultimodalRAGWithVideos/tests/test_multimodalragwithvideos_on_gaudi.sh

```diff
@@ -0,0 +1,272 @@
+#!/bin/bash
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+IMAGE_REPO=${IMAGE_REPO:-"opea"}
+IMAGE_TAG=${IMAGE_TAG:-"latest"}
+echo "REGISTRY=IMAGE_REPO=${IMAGE_REPO}"
+echo "TAG=IMAGE_TAG=${IMAGE_TAG}"
+export REGISTRY=${IMAGE_REPO}
+export TAG=${IMAGE_TAG}
+
+WORKPATH=$(dirname "$PWD")
+LOG_PATH="$WORKPATH/tests"
+ip_address=$(hostname -I | awk '{print $1}')
+
+export video_fn="WeAreGoingOnBullrun.mp4"
+
+function build_docker_images() {
+    cd $WORKPATH/docker_image_build
+    # git clone https://github.com/opea-project/GenAIComps.git
```

Comment:
We need this git clone in the test.

Reply:
Thanks for feedback. I have uncommented this. This was for testing before GenAIComp PRs were merged.

Reply:
@chensuyue  Please check again and click resolve conversation. Thanks

Reply:
@chensuyue May I click `resolve conversation` here? I think this is addressed successfully. Thanks for the feedback.